src/ngraph/runtime/gpu/gpu_external_function.cpp

```diff
@@ -481,13 +486,62 @@ using namespace std;
                 match_function_name = "func_" + node.get_name();
                 emitted_function.replace(offset, 5, match_function_name);
                 match_function_map.insert({match_function, match_function_name});
-                writer << emitted_function << "\n";
+                m_writer << emitted_function << "\n";
             }
-            node_function_map.insert({&node, match_function_name});
+            m_node_function_map.insert({&node, match_function_name});
         }
     }
+}
 
-    for (shared_ptr<Function> current_function : pass_manager.get_state().get_functions())
+void runtime::gpu::GPU_ExternalFunction::emit_allocate_temp_mem_pool(
```

Comment:
nit: emit_allocate_temp_mem_pool -> emit_temp_mem_pool_allocation / emit_function_memory_allocation

(If you change this one, then also change the corresponding release function below)